### ISSUE TITLE
"Remove" word typo

### ISSUE THREAD

**Issue by NefloDev** [madmustachecompany/Blast-Workout-App#89]
### Description

When going to remove an exercise from a workout the removal action on the verification pop up shows "Revome" instead of "Remove".
![Screenshot_20240805_232409_Blast](https://github.com/user-attachments/assets/acb68032-6af0-4201-8a98-fc7466871707)

### Reproduction steps

```bash
1. Go to an existing workout with at least one exercise.
2. Tap on edit workout [pencil icon]
3. Select an exercise
4. Tap on options [three dots icon]
5. Tap on remove exercise
6. See error on action with red text
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### OS

Android

**Comment by madmustachecompany** [comment]
@NefloDev 
Well spotted! 
We'll fix that in the next release, thanks 🙏

**Comment by madmustachecompany** [comment]
The issue should be resolved with version 1.73
